code/modules/cargo/bounties/assistant.dm

```diff
@@ -186,3 +186,26 @@
 	reward = 5000
 	required_count = 5
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/deadmouse)
+
+/datum/bounty/item/assistant/paper_bin
+	name = "Paper Bins"
+	description = "Our accounting division is all out of paper. We need a new shipment immediately."
+	reward = 2500
+	required_count = 5
+	wanted_types = list(/obj/item/paper_bin)
+
+
+/datum/bounty/item/assistant/crayons
+	name = "Crayon Boxes"
+	description = "Dr Jones' kid ate all our crayons again. Please send us yours."
+	reward = 3000
+	required_count = 5
+	wanted_types = list(/obj/item/storage/crayons)
```

Comment:
This isn't looking for crayons themselves, as the description implies, it's only looking for the crayon boxes.